conf/os_skills/system/en/en_0301_bahn.json

```diff
@@ -5,7 +5,7 @@
 		],
 		"process":[ {"type":"console", "expression":"SELECT description FROM bahn WHERE from='$1$' to='$2$';"}],
 		"actions":[ {"type":"answer", "select":"random", "phrases":[
-			"$description$",
```

Comment:
According to JSON specification there should not be trailing commas present in JSON

Reference - https://stackoverflow.com/a/201856